src/cmd/ksh93/tests/builtins.sh

```diff
@@ -1146,6 +1146,84 @@ fi
 EOF
 "$SHELL" -i "$sleepsig" 2> /dev/null || err_exit "'sleep -s' doesn't work with intervals of more than 30 seconds"
 
+# ======
+# integer
+# The number of seconds to sleep.
+SECONDS=0
+sleep 1
```

Comment:
Is this not covered by the test that follows? I am reluctant to add long-sleeping regression tests unless really necessary. Multiple one-second sleeps add up, so I shortened a bunch of them early on. Still, it already takes a long time to run the tests, and the longer they take, the less likely I am to have the time or patience to run them. So, fast regression tests help improve code quality.

Reply:
That test is redundant. I have removed it and have also shortened the sleep duration of the other tests.